table.go

```diff
@@ -129,6 +159,27 @@ type genTable[Obj any] struct {
 	primaryAnyIndexer    anyIndexer
 	secondaryAnyIndexers map[string]anyIndexer
 	indexPositions       map[string]int
+	acquiredInfo         atomic.Pointer[acquiredInfo]
+}
+
+func (t *genTable[Obj]) acquired(handle string, acquiredAt time.Time) {
+	t.acquiredInfo.Store(&acquiredInfo{
```

Comment:
This adds an extra allocation to the `WriteTxn` call per table. It doesn't seem to have a huge difference in benchmarks for 1 object per WriteTxn (few % slower) so I think it's worth the trade-off as the information this gives is quite useful.

Reply:
```
Name   Object count   Deleted objects   Indexes   Initializers   Go type      Last WriteTxn
ex     2              0                 id, x     [foo]          main.ExObj   example (1.2s ago, locked for 10.0us)
```
(the "Last WriteTxn" column is derived from this)

Reply:
Reimplemented this by storing the data in `txn` instead. Benchmarks back to previous numbers now.